### PR TITLE
Diona immune to FTL Knockdown

### PR DESCRIPTION
**Pull request by dvir001**
## About the PR
Give diona a stronger grip when getting hit by FTL jumps, allowing them to just stand normally without the need to sit.

## Why / Balance
Tree power

## How to test
N/A

## Media
N/A

## Requirements

- [X] I have read and am following the [Pull Request and Changelog Guidelines](https://docs.spacestation14.com/en/general-development/codebase-info/pull-request-guidelines.html).
- [X] I have added media to this PR or it does not require an ingame showcase.

## Breaking changes
N/A

**Changelog**
:cl:
- add: Diona are now immune to the force of an FTL jump.

### CODE REVIEW

File: Content.Server/Shuttles/Systems/ShuttleSystem.FasterThanLight.cs [hunk 1]
```diff
@@ -637,11 +637,9 @@ private void DoTheDinosaur(TransformComponent xform)
             {
                 if (!_statusQuery.TryGetComponent(child, out var status))
                     continue;
-                
-                if (HasComp<FTLKnockdownImmuneComponent>(child)) // Frontier: NPC knockdown immunity
-                    continue; // Frontier: NPC knockdown immunity
 
-                _stuns.TryParalyze(child, _hyperspaceKnockdownTime, true, status);
+                if (!HasComp<FTLKnockdownImmuneComponent>(child)) // Frontier: FTL knockdown immunity
+                    _stuns.TryParalyze(child, _hyperspaceKnockdownTime, true, status);
```
Comment (whatston3) [inline Content.Server/Shuttles/Systems/ShuttleSystem.FasterThanLight.cs:642]: Should FTL-immune players be tossed if standing on space?  Should NPCs?

Comment (dvir001) [reply Content.Server/Shuttles/Systems/ShuttleSystem.FasterThanLight.cs:642]: Yes and yes.
Stronger yeet.

**Comment by whatston3** [comment]
Works alright, not sure about the toss - in practice it means you were standing on lattice.

**Review by whatston3 — APPROVED** [review]
Generally seems fine, though I'll give way to stronger opinions.

**Comment by whatston3** [comment]
will add a guidebook mention

**Pull request merged**